### PR TITLE
ref task#4801: grammar detail activity

### PR DESCRIPTION
**Pull request by nguyenducanhit**
https://edu-redmine.framgia.vn/issues/4801

### CODE REVIEW

**Comment by nguyenducanhit** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -19,13 +19,9 @@
...
-            android:name=".screen.grammar.GrammarActivity"
-            android:theme="@style/AppTheme.NoActionBar"/>
-
+        <activity android:name=".screen.vocabulary.VocabularyDetailActivity" />
+        <activity android:name=".screen.grammar.GrammarActivity" />
+        <activity android:name=".screen.grammar_detail.GrammarDetailAcvitity" />
     </application>
 
-</manifest>
+</manifest>
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: add end of file

File: app/src/main/java/com/framgia/toeic/screen/grammar/GrammarActivity.java [hunk 2]
```diff
@@ -23,16 +23,27 @@ protected int getLayoutResource() {
...
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case android.R.id.home:
+                GrammarActivity.this.finish();
+                break;
+        }
+        return super.onOptionsItemSelected(item);
+    }
+
+    public static Intent getGrammarIntent(Context context) {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/toeic/screen/grammar/GrammarActivity.java]: https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md#28-class-member-ordering
convention 2.8

File: app/src/main/java/com/framgia/toeic/screen/grammar_detail/GrammarDetailAcvitity.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.grammar_detail, menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case android.R.id.home:
+                GrammarDetailAcvitity.this.finish();
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/toeic/screen/grammar_detail/GrammarDetailAcvitity.java]:
```suggestion
                finish();
```

File: app/src/main/res/layout/item_question_grammar.xml [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <RadioGroup
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:layout_marginLeft="8dp"
+        android:layout_marginEnd="8dp"
+        android:layout_marginRight="8dp"
```
Comment (DoanVanToan) [inline app/src/main/res/layout/item_question_grammar.xml]: hardcode

File: app/src/main/res/layout/item_question_grammar.xml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+            android:layout_marginBottom="@dimen/dp_8"
+            android:layout_weight="1" />
+    </RadioGroup>
+
+    <android.support.constraint.Guideline
+        android:id="@+id/guideline4"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintGuide_begin="124dp" />
```
Comment (DoanVanToan) [inline app/src/main/res/layout/item_question_grammar.xml]: hardcode
tại sao lại sử dụng guide line là 124dp

**Comment by nguyenducanhit** [comment]
ready

**Pull request merged**